src/foam/nanos/auth/AddressConfig.js

```diff
@@ -30,6 +34,17 @@
     {
       class: 'Int',
       name: 'suite'
+    },
+    {
+      class: 'Array',
+      of: 'foam.nanos.auth.ViewPropertyConfig',
+      name: 'propertyConfig'
```

Comment:
Could you add documentation:

Reply:
Agree, so this is just configuring the properties of the first row in address view. It should be limited to self.Address.STREET_NUMBER, self.Address.STREET_NAME, self.Address.SUITE and should mention that it only takes into account visibility hidden and order.

I wonder if you can try:
return this.propertyConfig.map( v => this.Address.getAxiomByName(v.propertyName).copyFrom(v));

Then somewhere between here and the address view you make 'var data = Address.properties = addressConfig.getProperties();
Then the view just displays data in a SectionedDetailView